code/__DEFINES/cargo.dm

```diff
@@ -32,4 +32,61 @@
     list("", "\improper S.T.E.A.L.T.H. pod MKVII", "A supply pod that, under normal circumstances, is completely invisible to conventional methods of detection. How are you even seeing this?"),\
     list("gondolapod", "gondola", "The silent walker. This one seems to be part of a delivery agency."),\
     list("", "", "")\
+)
+
+#define CONTRABAND_SYNDIE list(\
+	/obj/item/poster/random_contraband,\
+	/obj/item/reagent_containers/food/snacks/grown/cannabis,\
+	/obj/item/reagent_containers/food/snacks/grown/cannabis/rainbow,\
+	/obj/item/reagent_containers/food/snacks/grown/cannabis/white,\
+	/obj/item/storage/pill_bottle/zoom,\
+	/obj/item/storage/pill_bottle/happy,\
+	/obj/item/storage/pill_bottle/lsd,\
+	/obj/item/storage/pill_bottle/aranesp,\
+	/obj/item/storage/pill_bottle/stimulant,\
+	/obj/item/toy/cards/deck/syndicate,\
```

Comment:
These are a source of illegal technology, and also a potent weapon, as said above.
Tator only

Reply:
Looking into it,... that's a source of illegal tech via non-emagged Supply console?
That's been in the code for a while...

Reply:
Should I remove it altogether?

Reply:
Probably a maintainer question, but I believe we've been trying to move to getting illegal tech only from Traitor uplink purchased items to prevent cheese.

Reply:
Eh; I think it's safer to leave them out.